Examples/UIExplorer/StatusBarExample.js

```diff
@@ -293,6 +305,21 @@ exports.examples = [{
   },
 }];
 
+if (android) {
```

Comment:
I changed it here but I can't really use that in the code above since it's all part of a single example component. I'd rather keep it as a single example instead of one per prop since some props like animated affect others and are not useful by themselves.